lib/src/specs/flex/flex_widget.dart

```diff
@@ -1,11 +1,6 @@
 import 'package:flutter/widgets.dart';
 
-import '../../core/styled_widget.dart';
-import '../../factory/mix_provider.dart';
-import '../../factory/mix_provider_data.dart';
-import '../../widgets/gap_widget.dart';
-import '../container/box_widget.dart';
-import 'flex_spec.dart';
+import '../../../mix.dart';
```

Comment:
This is normal to happen on `VsCode`, but it's important that we don't import from the mix and keep all the relative imports. I wonder if there is a configuration to make this default

Reply:
Done 👍